src/touch.js

```diff
@@ -0,0 +1,89 @@
+var common = require('./common');
+var fs = require('fs');
+
+module.exports = _touch;
```

Comment:
Are you suggestiong that I move the exports below the definition of the `touch` function?